.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+    push:
+      branches: master
+    pull_request:
+      branches: master
+jobs:
+# This workflow contains a single job called "build"
+  ci:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-16.04
+
+    steps:
+    # To save time, detach and pull some of the largest docker files
+    - run: (docker pull node:11 && docker pull mongo:4.2.3-bionic) &
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '11.7.0'
+    # Create directories to put in web and server
+    - run: mkdir server
+    - run: mkdir web
+    # Checks-out tthe server repository under $GITHUB_WORKSPACE/server
+    - uses: actions/checkout@v2
+      with:
+        repository: 'upchieve/server'
+        path: 'server'
+    # Checks-out this web repository under $GITHUB_WORKSPACE/web
+    - uses: actions/checkout@v2
+      with:
+        path: 'web'
+    # Create an empty directory to use for the volume, and start the container
+    - run: cd server && cp config.example.js config.js && mkdir mongo-volume && docker-compose up -d
```

Comment:
The example config file doesn't have any volunteer codes set by default, but the e2e tests need to sign up volunteers with valid codes in order to pass. We'll need to find a way to get these tests to run without revealing production signup codes to the public.

Reply:
We can just add some default (fake) volunteer codes, right?